render-utils.js

```diff
@@ -69,4 +69,27 @@ export const renderArtists = (artist) => {
     artistContainer.append(artistNameEl, descriptionEl);
 
     return artistContainer;
+};
+
+export const renderBeatles = (beatle) => {
```

Comment:
seems like this should be renderBeatle, since it only renders one

Reply:
I used the plural because that function renders each Beatle, it just does it one at a time, but I understand what you mean. It does sound like it will render them all.